hw/ip/ascon/data/ascon.hjson

```diff
@@ -344,7 +344,7 @@
       swaccess: "rw",
       hwaccess: "hro",
       hwext:    "false"
-      hwqe:     "true"
+      hwqe:     "false"
```

Comment:
Please note that the title of this commit message is too long. It should not be more than 72 or 80 chars.